frontend/src/components/SelectApplication.vue

```diff
@@ -26,45 +37,49 @@ onMounted(async () => {
         return Promise.reject(error);
     }
 });
+
+function valueChanged(event: any) {
```

Comment:
The parameter of valueChanged function should be the selected environment, which is a string. And maybe we should give it a better name

function filterEnv(selectedEnv: string) {
}